### PR TITLE
migrate from sql server to mysql rds

### PR DESCRIPTION
**Pull request by averyschreiner**

### CODE REVIEW

File: Chefster.csproj [hunk 1]
```diff
@@ -36,6 +36,7 @@
       </PackageReference>
       <PackageReference Include="Microsoft.TestPlatform.TestHost" Version="17.10.0" />
       <PackageReference Include="MongoDB.Driver" Version="2.25.0" />
+      <PackageReference Include="Pomelo.EntityFrameworkCore.MySql" Version="8.0.2" />
```
Comment (Pchristy03) [inline Chefster.csproj:39]: Should this be `Microsoft.EntityFrameworkCore.MySql`?

File: Context/ChefsterDbContext.cs [hunk 2]
```diff
@@ -11,6 +11,14 @@ public class ChefsterDbContext(DbContextOptions<ChefsterDbContext> options) : Db
     public DbSet<ConsiderationsModel> Considerations { get; set; }
     public DbSet<PreviousRecipeModel> PreviousRecipes { get; set; }
 
+    string? endpoint = Environment.GetEnvironmentVariable("MYSQL_ENDPOINT");
+    string? db = Environment.GetEnvironmentVariable("MYSQL_DB");
+    string? username = Environment.GetEnvironmentVariable("MYSQL_USERNAME");
+    string? password = Environment.GetEnvironmentVariable("MYSQL_PASSWORD");
```
Comment (Pchristy03) [inline Context/ChefsterDbContext.cs]: Should we pass in the configuration here via dependency injection to get the variables?

File: Chefster.csproj (hunk 1 above)
Comment (averyschreiner) [reply Chefster.csproj:39]: Pomelo.EntityFrameworkCore.MySql seems to be one of the standards:
https://learn.microsoft.com/en-us/ef/core/providers/?tabs=dotnet-core-cli

Comment (Pchristy03) [reply Chefster.csproj:39]: Ope nvm then! 👍🏻

**Pull request merged**